### ISSUE TITLE
filterBestVotedImage returns null when all images have voteAverage 0

### ISSUE THREAD

**Issue by Fluxlicious** [php-tmdb/api#164]
There's a problem with the filter filterBestVotedImage(). If all the images it filter through have a voteAverage of 0, then it will return null whereas it should return the first occurrence of a 0 value.

Possible fix is to set the $voteAverage to -1 as initial value like this:
```
    public function filterBestVotedImage()
    {
        $currentImage = null;
        $voteAverage  = -1;

        /**
         * @var $image Image
         */
        foreach ($this->data as $image) {
            if ($image->getVoteAverage() > $voteAverage) {
                $voteAverage  = $image->getVoteAverage();
                $currentImage = $image;
            }
        }

        return $currentImage;
    }
```

**Comment by wtfzdotnet** [comment]
Would you be willing to submit a PR with a unit test? I have not been making any use of this library anymore myself and don't mind merging valid fixes, however I do not have the time anymore to dive in and debug this myself without any hints as to which tmdb id has these problems etc.

**Comment by Fluxlicious** [comment]
I'm quite new to github and i've never done a pull request. I tried, but i ran into problems with PHPStorm. Can't seem to find a fix right now.

**Comment by wtfzdotnet** [comment]
Closing as fixed per #175
